Dockerfile

```diff
@@ -15,4 +15,6 @@ RUN mv "$PHP_INI_DIR/php.ini-production" "$PHP_INI_DIR/php.ini" \
 	&& sed -ri -e 's/(MinProtocol\s*=\s*)TLSv1\.2/\1None/' /etc/ssl/openssl.cnf \
 	&& sed -ri -e 's/(CipherString\s*=\s*DEFAULT)@SECLEVEL=2/\1/' /etc/ssl/openssl.cnf
 
-COPY --chown=www-data:www-data ./ /app/
\ No newline at end of file
+COPY --chown=www-data:www-data ./ /app/
+
+CMD ["/app/start-rssbridge"]
```

Comment:
Missing newline. Also wouldn't it be better to rename this script to something like `docker-entrypoint.sh`, to indicate that it's related to Docker image?

Reply:
sure, no problem